### PR TITLE
DBZ-6424 Add docs for new JmxSignalChannel and JmxNotificationChannel

### PR DESCRIPTION
**Pull request by mfvitale**

### CODE REVIEW

**Review by jpechane — APPROVED** [review]
Technically LGTM, thanks!

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 1]
```diff
@@ -193,6 +194,51 @@ Key = `test_connector`
...
+| An array of comma-separated regular expressions that match the fully-qualified names of the tables to include in the snapshot. +
+Specify the names by using the same format as is required for the xref:{context}-property-signal-data-collection[signal.data.collection] configuration option.
+
+|`additional-condition`
+|_N/A_
+| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot.
+|===
+
+The following image shows how to send signal from `jconsole`
+
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:239]:
```suggestion
.Procedure
1. Use your preferred JMX client (for example. JConsole or JDK Mission Control) to connect to the MBean server.
2. Search for the Mbean `debezium.__<connector-type>__.management.signals.__<server>__`.
The Mbean exposes `signal` operations that accept the following input parameters:

p0:: The id of the signal.
p1:: The type of the signal, for example, `execute-snapshot`.
p2:: A JSON data field that contains additional information about the specified signal type.
+
Send an `execute-snapshot` signal by providing value for the input parameters. + In the JSON data field, include the information that is listed in the following table:
+
.Execute snapshot data fields
[cols="2,2,6",options="header"]
|===
|Field | Default | Value

|`type`
|`incremental`
| The type of the snapshot to run.
Currently {prodname} supports only the `incremental` type.

|`data-collections`
|_N/A_
| An array of comma-separated regular expressions that match the fully-qualified names of the tables to include in the snapshot. +
Specify the names by using the same format as is required for the xref:{context}-property-signal-data-collection[signal.data.collection] configuration option.

|`additional-condition`
|_N/A_
| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot.
|===
+
The following image shows an example of how to use JConsole to send a signal:
+
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
@mfvitale Completed my review. Let me know if you have any questions or if something doesn't look right to you. Thanks!

File: documentation/modules/ROOT/pages/configuration/notification.adoc [hunk 2]
```diff
@@ -183,8 +184,33 @@ By default, the following notification channels are available:
...
+
+Notifications are exposed in the bean named `debezium.<connector-type>.management.notifications.<server>` through the `Notifications` attribute.
+
+The image below shows the notification about the start of the incremental snapshot
+
+image::jmx-notification-attribute.png[JMX notification attribute]
+
+You can also discard the notifications by calling the `reset` operation on the bean.
+
+The notifications are also exposed as a JMX notification with type `debezium.notification`.
```
[the remark this answers is not in the document]
Comment (mfvitale) [reply documentation/modules/ROOT/pages/configuration/notification.adoc:212]: This is not something that you can decide to, notifications are automatically exposed as jmx notifications.

File: documentation/modules/ROOT/pages/configuration/notification.adoc [hunk 3]
```diff
@@ -183,8 +184,33 @@ By default, the following notification channels are available:
 
 * `sink`
 * `log`
+* `jmx`
 
- To use the `sink` notification channel, you must also set the `notification.sink.topic.name` configuration property to the name of the topic where you want {prodname} to send notifications.
+[IMPORTANT]
+====
+To use the `sink` notification channel, you must also set the `notification.sink.topic.name` configuration property to the name of the topic where you want {prodname} to send notifications.
```
[the remark this answers is not in the document]
Comment (mfvitale) [reply documentation/modules/ROOT/pages/configuration/notification.adoc:191]: Let me review this part.

[the remark this answers is not in the document]
Comment (mfvitale) [reply documentation/modules/ROOT/pages/configuration/notification.adoc:191]: I added a dedicated section as for Kafka signal properties. Also I noticed that the section about Kafka signal was still only for MySQL even if now it is available for all connectors, so I moved it.

File: documentation/modules/ROOT/pages/configuration/signalling.adoc [hunk 4]
```diff
@@ -193,6 +194,52 @@ Key = `test_connector`
...
+1. Use your preferred JMX client (for example. JConsole or JDK Mission Control) to connect to the MBean server.
+2. Search for the Mbean `debezium.__<connector-type>__.management.signals.__<server>__`.
+The Mbean exposes `signal` operations that accept the following input parameters:
+
+p0:: The id of the signal.
+p1:: The type of the signal, for example, `execute-snapshot`.
+p2:: A JSON data field that contains additional information about the specified signal type.
++
+Send an `execute-snapshot` signal by providing value for the input parameters. + In the JSON data field, include the information that is listed in the following table:
++
```
Comment (roldanbob) [inline documentation/modules/ROOT/pages/configuration/signalling.adoc:218]:
```suggestion
3. Send an `execute-snapshot` signal by providing value for the input parameters. +
In the JSON data field, include the information that is listed in the following table:
+
```

**Review by roldanbob — CHANGES_REQUESTED** [review]
A handful of additional suggestions. After you make your next round of changes, I'll do a local Antora build to see how things look and get a better sense of whether we've missed anything and verify that the formatting is good.

**Comment by roldanbob** [comment]
> I think we are ready to merge @roldanbob

Yes, I just checked the rendering, and everything LGTM. Thanks!

**Pull request merged**